app/src/main/java/org/mozilla/focus/menu/BlockingItemViewHolder.java

```diff
@@ -49,6 +53,15 @@ public void run() {
         });
     }
 
+    private void disableTrackingCount(final TextView view) {
+        ThreadUtils.postToMainThread(new Runnable() {
+            @Override
+            public void run() {
+                view.setText("-");
```

Comment:
I wonder if we'd ever want to show something different in certain locales - but I guess we'll get feedback if that's needed!